Makefile

```diff
@@ -128,6 +128,11 @@ test: GOTEST_PKGS := $(foreach pkg,$(filter-out $(GOTEST_EXCLUDE),$(wildcard */)
 test: ## Run unit tests.
 	$(GO) run github.com/onsi/ginkgo/v2/ginkgo -v -p -randomize-all --tags static $(GOTEST_PKGS)
 
+
+.PHONY: e2e-test
+e2e-test: $(BIN) ## Run CLI end-to-end tests.
+	$(GO) run github.com/onsi/ginkgo/v2/ginkgo -v -p -randomize-all test/e2e/cli/...
+
```

Comment:
I would use `test-*` as the prefix to keep things consistent amongst existing target names (i.e. `buildenvs-*`) and I would create a global target to test both units and e2e or allow to call each one individually:

```diff
+ test: test-unit
+ test: test-e2e
+ test: ## Run all tests.
[...]
- test: ## Run unit tests.
+ test-unit: ## Run unit tests.
[...]
- e2e-test: $(BIN) ## Run CLI end-to-end tests.
+ test-e2e: $(BIN) ## Run CLI end-to-end tests.
```

Reply:
- Renaming done in [fd436233..efa95e3b](https://github.com/unikraft/kraftkit/compare/fd436233cc697f83f06dcf3fc86a813e996c45d1..efa95e3b4c47bf5ced602c4dc8a0adf68b34bb16)
- Global goal in 9b8c2457

Reply:
`make help` no longer shows the `test` target:
```
KraftKit developer build targets.

USAGE
  make [VAR=... [VAR=...]] TARGET

TARGETS
  devenv                   Start the development environment container.
  tidy                     Tidy import Go modules.
  fmt                      Format all files according to linting preferences.
  cicheck                  Run CI checks.
  test-unit                Run unit tests.
  test-e2e                 Run CLI end-to-end tests.
  install-golangci-lint    Install the Golang CI lint tool
  properclean              Completely clean the repository's build artifacts.
  help                     Show this help menu and exit.
  buildenv-base            OCI image used for building Unikraft unikernels with kraft.
  buildenv-gcc             OCI image containing a Unikraft-centric build of gcc.
  buildenv-myself-full     OCI image containing the build environment for KraftKit.
  buildenv-myself          OCI image containing KraftKit binaries.
  buildenv-qemu            OCI image containing a Unikraft-centric build of QEMU.
  kraft                    The kraft binary.
  ```

Reply:
I forgot one hash mark apparently, nice catch!
Fixed in [9b8c2457..bd2f84ef](https://github.com/unikraft/kraftkit/compare/9b8c2457817f06190eed09d918bf5f70e46229eb..bd2f84ef63bac073d587184ec2f1294715231021).